Source/JavaScriptCore/runtime/DateConstructor.cpp

```diff
@@ -92,7 +92,7 @@ static inline double makeTime(double hour, double min, double sec, double ms)
 }
 
 // https://tc39.es/ecma262/#sec-makeday
-static inline double makeDay(double year, double month, double date)
+double makeDay(double year, double month, double date)
```

Comment:
This is not great. Put it into header is you would like to use it. Not making it non-inline.

Reply:
Fixed